pantheon.php

```diff
@@ -40,6 +40,11 @@
 		 */
 		define( 'FS_METHOD', 'direct' );
 	}
+    // When developing a WordPress Multisite locally, ensure that this constant is set.
+    // This will set the Multisite variable in all Pantheon environments.
+    if ( getenv( 'FRAMEWORK' ) == 'wordpress_network' && !defined( 'WP_ALLOW_MULTISITE' ) ) {
```

Comment:
@pantheon-systems/cms-platform Should we include the erroneous framework value `wordpress_multisite` as well?

Reply:
No. 

This is designed to make things easier for WordPress multisites on the platform. Using an old/erroneous framework value would _not_ be an easier experience, so we shouldn't put time or value into supporting unsupported/bad packages.

Also, this is a value that is mostly relevant when first setting up/creating a WP multisite and it's not possible (outside of a custom upstream that was created a long time ago) to create a new multisite using a `wordpress_multisite` framework.

The ONLY argument for supporting `wordpress_multisite` frameworks would be if we wanted to remove the `wordpress_network` language in favor of "multisite", but that's not a problem I think we're likely to want to take on.